Project.toml

```
@@ -20,8 +20,8 @@ Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 [compat]
 CUDA = "^1.3, ^2, ^3"
 DocStringExtensions = "0.8, 0.9"
-FourierFlows = "0.9.2"
-GeophysicalFlows = "0.14"
+FourierFlows = "0.10.0"
+GeophysicalFlows = "0.15"
```

```suggestion
GeophysicalFlows = "0.15.0”
```

not sure if it matters though